test/core/Rectangle.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-describe('PIXI.Rectangle', function ()
+describe('PIXI.Rectangle', () =>
 {
-    it('should create a new rectangle', function ()
+    it('should create a new rectangle', () =>
```

Comment:
I can certainly change them back to `function`--I just saw the arrow functions in some other tests (e.g., `Graphics.js`, `Container.js`, `DisplayObject.js`) and assumed it was OK.

I can go through and change other tests back to the `function` syntax in another PR if you'd like.